src/actions/sources.js

```diff
@@ -145,9 +152,8 @@ function loadSourceMap(generatedSource) {
 
     dispatch({ type: "ADD_SOURCES", sources: originalSources });
 
-    await dispatch(loadSourceText(generatedSource));
```

Comment:
@codehag in another PR i mentioned that we should leave this line in because I was removing it here :)

In this PR, as it is now, we can remove this because we load it if we're syncing pending BPs for a source. We might not need it there either!

If we discover that we do need it, and for some reasons our tests pass without it, we should update our tests!